File: scratch/query_payments.js
const Database = require('better-sqlite3');
const path = require('path');

try {
  const db = new Database('C:\\Users\\Raman Duggal\\OneDrive\\BIZSUITE\\season\\2\\jrmd.sqlite', {readonly: true});
  console.log("Database opened successfully.");
  
  const tables = ['payments'];
  for (const t of tables) {
    try {
      const results = db.prepare(`SELECT id, data FROM ${t}`).all();
      console.log(`Checking table ${t}, found ${results.length} rows...`);
      for (const row of results) {
        const data = JSON.parse(row.data);
        if (data.bankAcNo === '32501169461' || data.amount === 230510 || data.rtgsAmount === 230510) {
          console.log(`FOUND in ${t}:`, JSON.stringify(data, null, 2));
        }
      }
    } catch(e) {
      console.log(`Error checking table ${t}:`, e.message);
    }
  }
} catch(e) {
  console.log("DB Open Error:", e);
}

File: scratch/query_payments.py
import sqlite3
import json

try:
    db_path = r'C:\Users\Raman Duggal\OneDrive\BIZSUITE\season\2\jrmd.sqlite'
    conn = sqlite3.connect(db_path)
    cursor = conn.cursor()
    cursor.execute("SELECT id, data FROM payments")
    rows = cursor.fetchall()
    print(f"Total payments row count: {len(rows)}")
    for row in rows:
        data = json.loads(row[1])
        ac = data.get('bankAcNo', '')
        amt = data.get('amount', 0)
        rtgs_amt = data.get('rtgsAmount', 0)
        if ac == '32501169461' or amt == 230510 or rtgs_amt == 230510:
            print("FOUND PAYMENT:", json.dumps(data, indent=2))
except Exception as e:
    print("ERROR:", e)
